### DISCUSSION TITLE
Multiple classes for width 50% within typography

### DISCUSSION THREAD

**Discussion by marko-mitrovic** [tailwindlabs/tailwindcss#14874] [Help]
Hello,

I would just like to ask what is the point of having two classes within typography when we are talking about 50% width, `w-1/2` and `w-6/12`. Actually, when we should use `w-6/12` and what's the point of that other one?

**Comment by robbyrice** [comment]
This allows you to have the following HTML.

````
<div>
  <div class="w-1/2">
  </div>
  <div class="w-1/2">
  </div>
</div>

<div>
  <div class="w-6/12">
  </div>
  <div class="w-3/12">
  </div>
  <div class="w-3/12">
  </div>
</div>
````

Or something similar.
